packages/cli/src/controller/codegen-cosmos.test.ts

```diff
@@ -32,6 +30,9 @@ describe('Able to generate cosmos types from protobuf', () => {
     await promisify(rimraf)(path.join(__dirname, '../../test/protoTest1/src'));
   });
 
+  const cosmosModule = loadDependency(NETWORK_FAMILY.cosmos);
+  const manifest = loadFromJsonOrYaml(path.join(PROJECT_PATH, 'project.yaml')) as ProjectManifestV1_0_0<any, any, any>;
```

Comment:
Does this work? It looks like there are default generics

```suggestion
  const manifest = loadFromJsonOrYaml(path.join(PROJECT_PATH, 'project.yaml')) as ProjectManifestV1_0_0;
```